staticaddr/withdraw/manager.go

```diff
@@ -380,32 +383,49 @@ func (m *Manager) createFinalizedWithdrawalTx(ctx context.Context,
 		).FeePerKWeight()
 	}
 
-	outpoints := toOutpoints(deposits)
-	resp, err := m.cfg.StaticAddressServerClient.ServerWithdrawDeposits(
-		ctx, &staticaddressrpc.ServerWithdrawRequest{
-			Outpoints:       toPrevoutInfo(outpoints),
-			ClientNonces:    clientNonces,
-			ClientSweepAddr: withdrawalAddress.String(),
-			TxFeeRate:       uint64(withdrawalSweepFeeRate),
-		},
+	params, err := m.cfg.AddressManager.GetStaticAddressParameters(
+		ctx,
 	)
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("couldn't get confirmation height for "+
+			"deposit, %w", err)
 	}
 
-	addressParams, err := m.cfg.AddressManager.GetStaticAddressParameters(
-		ctx,
+	// Send change back to the static address.
+	changeAddress, err := m.cfg.AddressManager.GetTaprootAddress(
```

Comment:
Why `m.cfg.AddressManager.GetStaticAddress(ctx)` can't be used here?

Reply:
Good catch, I changed it.